packages/modules/src/SessionKeyManagerModule.ts

```diff
@@ -131,7 +145,11 @@ export class SessionKeyManagerModule extends BaseValidationModule {
    * @param sessionSigner The signer to be used to sign the user operation
    * @returns The signature of the user operation
    */
-  async signUserOpHash(userOpHash: string, sessionParam?: SessionParams): Promise<string> {
+  async signUserOpHash(userOpHash: string, sessionParams?: SessionParams[]): Promise<string> {
+    if (!sessionParams || sessionParams.length === 0 || sessionParams.length > 1) {
+      throw new Error('Session parameters are not provided or invalid')
```

Comment:
Error message can be more clear in terms of requirements 

`Session parameters are not provided or of invalid length. Only one object is allowed in sessionParams array.`

Reply:
ok

Reply:
done